PubNub/Core/PubNub+Core.h

```diff
@@ -50,6 +50,15 @@
  */
 - (NSString *)uuid;
 
+/**
+ @brief     Retrieve subscribe loop filter expression set during client initialization
+ 
+ @return User provided filter expression or nil if none set
+ 
+ @since 4.3
+ */
+- (NSString *)filterExpression;
+
```

Comment:
@jzucker2 Is there any requirement by specification to provide access to configured filter expression? If there is such requirement, more reasonable place for this in _PubNub+Subscribe_ category file.